performance/src_ios/fake/FIRHTTPMetric.h

```diff
@@ -5,18 +5,19 @@
 /* Different HTTP methods. */
 typedef NS_ENUM(NSInteger, FIRHTTPMethod) {
   FIRHTTPMethodGET NS_SWIFT_NAME(get),
-  FIRHTTPMethodPUT NS_SWIFT_NAME(put),
-  FIRHTTPMethodPOST NS_SWIFT_NAME(post),
-  FIRHTTPMethodDELETE NS_SWIFT_NAME(delete),
-  FIRHTTPMethodHEAD NS_SWIFT_NAME(head),
-  FIRHTTPMethodPATCH NS_SWIFT_NAME(patch),
-  FIRHTTPMethodOPTIONS NS_SWIFT_NAME(options),
-  FIRHTTPMethodTRACE NS_SWIFT_NAME(trace),
-  FIRHTTPMethodCONNECT NS_SWIFT_NAME(connect)
-} NS_SWIFT_NAME(HTTPMethod);
+      FIRHTTPMethodPUT NS_SWIFT_NAME(put),
```

Comment:
hahaha what is going on here.

(probably a side effect of this being an ObjC file)